lib/configure.js

```diff
@@ -287,6 +288,10 @@ function configure (gyp, argv, callback) {
         argv.push('-Dnode_exp_file=' + node_exp_file)
       }
       argv.push('-Dnode_gyp_dir=' + nodeGypDir)
+
+      // Do this to keep Cygwin environments happy, else the unescaped '\' gets eaten up,
+      // resulting in bad paths, Ex c:parentFolderfolderanotherFolder instead of c:\parentFolder\folder\anotherFolder
+      nodeLibFile = nodeLibFile.replace(/\\/g, '\\\\')
```

Comment:
I don't think this is a windows-only branch so to be safe maybe this should only be invoked if windows

Reply:
Do posix paths contain backslashes?

Reply:
```
$ touch 'foo\\bar'
$ ls -al foo\\\\bar
-rw-r--r-- 1 rvagg staff 0 Jul 15 16:56 'foo\\bar'
```

Reply:
This still needs to be addressed @joquijada. I think can you wrap this in a conditional for Cygwin or at least Windows

Reply:
@rvagg Done, wrapped in a Windows conditional.